packages/server/src/api/todo/adx/account.ts

```diff
@@ -49,7 +50,7 @@ export default function (server: Server) {
         .select([
           'invite.disabled as disabled',
           'invite.availableUses as availableUses',
-          sql<number>`count(code_use.usedBy)`.as('useCount'),
+          sql<number>`count(${ref('code_use.usedBy')})`.as('useCount'),
```

Comment:
is `ref` only required here because the column name is case-sensitive?

we can always change column names to snake case, wouldn't be a big deal 🧐

Reply:
Yep, I believe that is the case.  I have gotten sort of used to this kind of thing from working with objection, so I don't have a strong opinion on it either way!